spec/features/publish/courses/editing_subject_spec.rb

```diff
@@ -80,24 +146,43 @@ def then_i_am_met_with_course_details_page
     expect(page).to have_current_path("/publish/organisations/#{provider.provider_code}/#{Settings.current_recruitment_cycle_year}/courses/#{course.course_code}/details")
   end
 
-  def then_i_am_met_with_the_modern_languages_page
-    expect(page).to have_current_path("/publish/organisations/#{provider.provider_code}/#{Settings.current_recruitment_cycle_year}/courses/#{course.course_code}/modern-languages?#{params_with_subject}")
+  def then_i_am_met_with_the_modern_languages_page(extra_subject = nil)
+    expect(page).to have_current_path(
+      [
+        "/publish/organisations/#{provider.provider_code}/",
+        "#{Settings.current_recruitment_cycle_year}/",
+        "courses/#{course.course_code}/modern-languages?",
+        params_with_subject(extra_subject).to_s
+      ].join
+    )
     expect(page).to have_content('Languages')
   end
 
+  def then_i_am_met_with_an_error
+    within('.govuk-error-summary') do
+      expect(page).to have_content('Select a subject')
```

Comment:
This is nitpicking, but I think we should make sure we see the error in the summary and on the input -- So could do `expect(page).to have_content('Select a subject).twice` Or you can be more specific and check it is in the error summary as on the input field.

Reply:
Good shout, I'll remember to do that in future